compiler/src/dotty/tools/backend/jvm/GenBCode.scala

```diff
@@ -275,7 +275,8 @@ class GenBCodePipeline(val int: DottyBackendInterface, val primitives: DottyPrim
             catch case ex: ClosedByInterruptException =>
               try
                 outTastyFile.delete() // don't leave an empty or half-written tastyfile around after an interrupt
-              catch case _: Throwable =>
+              catch
+                case _: Throwable =>
```

Comment:
also `catch case ioe: Throwable => ex.addSuppressed(ioe)`, I don't see how to suggest change on multiple lines.

Maybe it shouldn't catch Throwable, but at least it rethrows immediately.